src/invoke-lambda/run-in-node.js

```diff
@@ -1,15 +1,19 @@
-let fs = require('fs')
-let path = require('path')
+let { readFile } = require('fs')
+let { join } = require('path')
 let spawn = require('./spawn')
 
-module.exports = function runInNode (options, request, timeout, callback) {
-  let node = path.join(__dirname, 'runtimes', 'node.js')
-  fs.readFile(node, 'utf8', function done (err, data) {
+module.exports = function runInNode (params, callback) {
+  let node = join(__dirname, 'runtimes', 'node.js')
+  readFile(node, 'utf8', function done (err, data) {
```

Comment:
While we're here, why not read the node runtime executor once, perhaps in this module's global scope? Instead of doing so on every invocation?

Reply:
Prob won't be discernible improvement but why not, no reason not to!